core/src/G3Reader.cxx

```diff
@@ -100,14 +102,17 @@ void G3Reader::Process(G3FramePtr frame, std::deque<G3FramePtr> &out)
 }
 
 off_t G3Reader::Seek(off_t offset) {
-	try {
-		return g3_istream_seek(stream_, offset);
-	} catch (...) {
-		log_fatal("Cannot seek %s; stream closed at EOF.", cur_file_.c_str());
-	}
+	return g3_istream_seek(stream_, offset);
+	// try {
+	// 	return g3_istream_seek(stream_, offset);
+	// } catch (...) {
+	// 	log_fatal("Cannot seek %s; stream closed at EOF.", cur_file_.c_str());
+	// }
```

Comment:
Presumably this being commented out isn't intentional? Can the code just be deleted?

Reply:
Ah, that was for debugging.  I've replaced it with a similar error to `tell()` that requires `counter=True`.